fda-model/metrics/ExpectedAnnualDamageResults.cs

```diff
@@ -18,15 +18,12 @@ public void AddEADEstimate(double eadEstimate, string category)
         {
             double[] data = new double[1] { eadEstimate };
             IData ead = IDataFactory.Factory(data);
-            if (_ead.ContainsKey(category))
+            if (!_ead.ContainsKey(category))
             {
-                _ead[category].AddObservationToHistogram(ead); 
-            }
-            else
-            {
-                var histo = new Histogram(ead, EAD_HISTOGRAM_BINWIDTH);
+                var histo = new Histogram(null, EAD_HISTOGRAM_BINWIDTH);
```

Comment:
having a null passed in a constructor is kinda bad form. why is this an improvement from the previous method of passing in an idata with a single estimate?

Reply:
I was having problems initiating a histogram with 1 observation. I identify the number of bins as (max - min)/binwidth. Initiating a histogram with 1 observation makes that ratio 0. 

I suppose I could add logic along the lines of if (data.count == 1) { make a 1 bin histogram } ?

Reply:
yeah a one bin either centered or starting on the observation